### DISCUSSION TITLE
cant connect to my account

### DISCUSSION THREAD

**Discussion by Alexnoj** [requarks/wiki#4594] [Help / Questions] [answered]
I just install wikijs self host win10 with PostgreSQL 13.
when I try connect I got error. I write the right email/pass.
I tried reset the admin password but i dont understand how edit the value bcrypt-formatted in PostgreSQL,

**Comment by NGPixel** [comment, marked as answer]
You can use https://bcrypt-generator.com/ to generate a new bcrypt value.

**Reply by Alexnoj** [reply]
yes I saw it in the guide. The problem is I dont how edit the SQL.
it is happen because I leave blank volume WEB SITE when I installed wikijs?

**Reply by NGPixel** [reply]
During setup, did you write your email in lowercase?

**Reply by Alexnoj** [reply]
no. I will try reinstall.
Ty

**Reply by Alexnoj** [reply]
> During setup, did you write your email in lowercase?

its work. Thanks.

one more question.
I use WYSIWYG editor How can I do sub numbered list ike:
1. vvdksadasio
2. dsadsadsadas
3. dsadsadas
4.  gfgfdgfdgfd:
     1.dsadsa
     2.dsadasdasdas
5.dasdsadasdas
